schema/meter_provider.json

```diff
@@ -186,6 +185,116 @@
             ],
             "title": "OtlpMetric"
         },
+        "AggregationTemporalityPreferenceByInstrumentKind": {
+            "type": ["object", "null"],
+            "additionalProperties": false,
+            "properties": {
+                "counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "histogram": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "observable_counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "observable_gauge": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "observable_up_down_counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "up_down_counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                }
+            }
+        },
+        "AggregationTemporalityPreference": {
```

Comment:
The spec says that aggregation temporality should be configurable as a function of instrument kind, but most users will want to use these [shorthand options](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#additional-environment-variable-configuration). 

This is much simpler / more compact than writing the preference for each instrument kind. In the future, we can add a separate property to give full control over temporality by instrument kind if needed.